osquery/distributed/distributed.cpp

```diff
@@ -120,6 +120,7 @@ void Distributed::addResult(const DistributedQueryResult& result) {
 
 Status Distributed::runQueries() {
   while (getPendingQueryCount() > 0) {
+    VLOG(1) << "Executing distributed query: " << query.query;
```

Comment:
`query` is not available here, move this below line 124. You may also want the name of the query too?
```cpp
VLOG(1) << "Executing distributed query (" << query.id << "): " << query.query;
```